cs6232-g4/View/ViewReturnHistoryForm.cs

```diff
@@ -0,0 +1,78 @@
+﻿using cs6232_g4.Model;
+using Furnitures.Model;
+using Members.Controller;
+using System.Windows.Forms;
+
+namespace cs6232_g4.View
+{
+    public partial class ViewReturnHistoryForm : Form
+    {
+        /// <summary>
+        /// Initializes a new instance of the View Return History class.
+        /// </summary>
+        /// 
+
+        private readonly TransactionController _transactionController;
+        private List<ReturnTransaction> returns;
+        private int MemberID;
+        private string MemberName;
+        private BindingSource transactionDALBindingSource;
+        private List<ReturnLineItem> returnLineItemList;
+        public ViewReturnHistoryForm(int mbrID, string mbrName)
+        {
+            InitializeComponent();
+            _transactionController = new TransactionController();
+            returns = new List<ReturnTransaction>();
+            this.returnLineItemList = new List<ReturnLineItem>();
+            this.MemberID = mbrID;
+            this.MemberName = mbrName;
+            transactionDALBindingSource = new BindingSource();
+        }
+
+        private void ViewReturnHistory_Load(object sender, EventArgs e)
```

Comment:
this function isn't hooked to the form to trigger. Refer to the screenshot below
![Screenshot 2024-04-17 at 8 35 53 PM](https://github.com/lk00044/MLJFurnitureStore/assets/112569246/3044ce83-e0f1-4561-a29d-1ec0b6df9b7a)

Also, after I hooked it, I got the following error, it seems that the queries aren't working. I suggest you make sure the queries are working first on a sql script then implement them in VS once you verify data looks good
![Screenshot 2024-04-17 at 8 35 39 PM](https://github.com/lk00044/MLJFurnitureStore/assets/112569246/84a39314-5d73-4c8c-a067-4a88418efc20)